src/main.js

```diff
@@ -1145,10 +1145,7 @@ function getDeeplinkingURL(args) {
 }
 
 function shouldShowTrayIcon() {
-  if (process.platform === 'win32') {
-    return true;
-  }
-  if (['darwin', 'linux'].includes(process.platform) && config.showTrayIcon === true) {
+  if (config.showTrayIcon === true) {
```

Comment:
same here

Reply:
Doesn't this now include all operating systems? Since the only needed condition is that `showTrayIcon` is true?

Reply:
but having the default to true would be different than forcing win32 to be true, right? so it should be something like:
```suggestion
  if (config.showTrayIcon === true || process.platform === 'win32') {
```